translations/ui-agreements/en.json

```diff
@@ -484,11 +484,15 @@
 
   "settings.actions": "Actions",
   "settings.general": "General",
+  "settings.displaySettings": "Display settings",
   "settings.general.hideEResourcesFunctionality.title": "Hide internal agreements knowledgebase",
   "settings.general.hideEResourcesFunctionality.description": "If you are using another app (e.g. eHoldings) as your knowledge base and wish to hide the internal agreements knowledge base, you can check this setting. Specifically, this will:",
   "settings.hideEResources.result.1": "Hide the E-resources tab in the agreements app",
   "settings.hideEResources.result.2": "Hide the basket icon in the agreements app",
   "settings.hideEResources.result.3": "Hide the \"E-resources covered by this agreement\" accordion in individual agreements",
+  "settings.general.displaySuppressFromDiscovery.description": "Display \"Suppress from discovery\" on",
```

Comment:
Suppress lower case? so `suppress`

Reply:
Here I did this because it's in quotes, and so refers to the field itself, which is `"Suppress from discovery"` I feel like outside of the quotes it would have to be sentence case but inside them it should be uppercased. I can ask Gill for clarification though if you'd like.

Reply:
I was also following Gill's wireframe, but I've asked her for clarification on this.